config/config.exs

```diff
@@ -14,6 +14,16 @@ config :naboo, Naboo.Endpoint,
 
 config :naboo, Naboo.Repo, start_apps_before_migration: [:ssl]
 
+config :phoenix_swagger, json_library: Jason
+
+config :naboo, :phoenix_swagger,
+  swagger_files: %{
+    "priv/static/swagger.json" => [
+      endpoint: Naboo.Endpoint,
+      router: NabooAPI.Router.Swagger
```

Comment:
Shouldn't this router be `NabooAPI.Router.Urls`

Reply:
?

Reply:
the swagger route is defined in the module NabooAPI.Router.Swagger inside swagger.ex

Reply:
I made the change [here](https://github.com/ARKultur/naboo/commit/f3f64144afa46f0c9c10e3583996a80ba8d500be#diff-fcb6e9bf2546587075d6850070c1b4d26f061f0c199e6ea3ac4d8fd76d9fa8f6L38) so it would work. Did you test it ?